src/control/Control.cpp

```diff
@@ -1923,70 +1920,60 @@ auto Control::newFile(string pageTemplate, Path fileName) -> bool {
 /**
  * Check if this is an autosave file, return false in this case and display a user instruction
  */
-auto Control::shouldFileOpen(string filename) -> bool {
-    // Compare case insensitive, just in case (Windows, FAT Filesystem etc.)
-
-    filename = StringUtils::toLowerCase(filename);
-    string basename = StringUtils::toLowerCase(Util::getConfigSubfolder("").str());
+auto Control::shouldFileOpen(fs::path const& filepath) const -> bool {
+    auto basePath = Util::getConfigSubfolder("");
 
-    if (basename.size() > filename.size()) {
+    if (basePath < filepath) {
         return true;
     }
```

Comment:
I don't think, this is not a good way to test if this may be a subdirectory or not, since those paths are lexicographically compared.
```  suggestion
```